projects/manager/src/app/provenance/provenance.component.html

```diff
@@ -0,0 +1,28 @@
+<form [formGroup]="form">
+    <div formArrayName="provenance" *ngFor="let control of form.controls.provenance.controls; let i = index">
+        <mat-divider *ngIf="i > 0"></mat-divider>
```

Comment:
Maybe use new for/if syntax?
You could also use `!$first` instead of i > 0?

Or something like
`@for (item of items; track item; let notFirst = !$first) {`

Reply:
I think it's very concise as it is.